admin/spam/views.py

```diff
@@ -126,23 +159,18 @@ def get_queryset(self):
         return Comment.find(query).sort(self.ordering)
 
     def get_context_data(self, **kwargs):
-        queryset = kwargs.pop('object_list', self.object_list)
```

Comment:
Although I think this would be a good idea, it should be for a future PR. Although not as concise as possible and with a bit of repetition I think it is not too terrible. Really this is a request for using the API serializers which I might explore a bit more after the main section of this is done.